apollo-relay/README.md

```diff
@@ -0,0 +1,77 @@
+# Apollo Telemetry
+Relay statistics from router to apollo studio ingress.
+
+## Authentication
+
+The server must be authenticated to submit messages to the Apollo ingress.
+Furthermore, the statistics must be submitted "to" a particular graph. In
+the existing products this is accomplished using [environment variables](https://www.apollographql.com/docs/federation/managed-federation/setup/#4-connect-the-gateway-to-studio).
+
+In the router, we have a configuration file which can be dynamically
+re-loaded, so it makes more sense to include this configuration here. There
+is a new optional section that looks like this:
+
+```
+graph:
```

Comment:
Unless you have a strong reason not to, it would probably be good to match the naming scheme used in Apollo Server (which are relatively recently chosen names, not too old and janky yet).

```
apollo:
  key: 
  graphRef:
```

Reply:
I do prefer "reference" to "graphRef", but it's not a strongly held preference.

Reply:
I too prefer avoiding unnecessary abbreviations but I think ref is the term we've mostly settled on.